schema/xml/metaschema.xsd

```diff
@@ -860,6 +860,12 @@
             that will prevent typical use of the content.</xs:documentation>
         </xs:annotation>
       </xs:enumeration>
+      <xs:enumeration value="DEBUG">
+        <xs:annotation>
+          <xs:documentation>A violation of the constraint represents a fault in the content that
+            warrants review by a developer when performing model or tool development.</xs:documentation>
+        </xs:annotation>
+      </xs:enumeration>      
```

Comment:
Move this to after INFORMATIONAL, since these are listed in order of severity?

Reply:
Relocated as requested in https://github.com/usnistgov/metaschema/commit/6ac24990042fded121843e8ee9e51a4b988e71bb.